ValheimPlus/Configurations/Configuration.cs

```diff
@@ -31,5 +31,7 @@ public class Configuration
         public CameraConfiguration Camera { get; set; }
         public GameConfiguration Game { get; set; }
         public VagonConfiguration Wagon { get; set; }
+        public SleepConfiguration Sleep { get; set; }
+        public UnarmedConfiguration Unarmed { get; set; }
```

Comment:
I think for now we should keep unarmed configuration where it is until we have a large discussion about our convention on organizing settings

Reply:
I disagree, placing it under player with an incorrect description doesn't seem correct to me.  It doesn't modify the base unarmed damage and was moved without that same discussion, I am moving it back to where I originally implemented it.

Reply:
I agree with @GaelicGamer, I was going to create an issue to request that it be moved somewhere else for people like me who want to modify the Player section but don't want to modify Unarmed Damage.

Reply:
Unarmed damage is a property of Player

If you don't want to modify Unarmed damage then simply do not modify it

The same way that you can set hotkeys and values for other settings, unarmed damage is but a property of the Player

Furthermore, its code is used in the Humanoid, if ever we are to add any other modifier to Human, do we then create a setting for each? Jump distance ? Run speed ?

At most, a separate config for PlayerModifiers

But a whole setting class only for Unarmed makes no sense

Reply:
> 
> 
> Unarmed damage is a property of Player
> 
> If you don't want to modify Unarmed damage then simply do not modify it
> 
> The same way that you can set hotkeys and values for other settings, unarmed damage is but a property of the Player
> 
> Furthermore, its code is used in the Humanoid, if ever we are to add any other modifier to Human, do we then create a setting for each? Jump distance ? Run speed ?
> 
> At most, a separate config for PlayerModifiers
> 
> But a whole setting class only for Unarmed makes no sense

Correct me if I'm wrong but your implementation actually modifies the way Unarmed Damage works, doesn't it?
At the very least the default value shipped with Valheim Plus' config is not the same as the base game's, I noticed a considerable difference when you guys implemented it. Which is to say there is no way right now to "not use it".